src/index.js

```diff
@@ -1,48 +1,103 @@
 import { handleDragStart, handleDrop, cancelDefault } from './services/sort';
-import { updateStatus, getTodos } from './services/update';
+import {
+  updateStatus, getTodos, saveToStorage, orderAndSave,
+} from './services/update';
+import {
+  createTodo, clearAll, clearList, editTodoInStorage,
+  editTodoInList, deleteTodoFromStorage, deleteTodoFromList,
+} from './services/todoService';
 import './style.css';
 
 class TodoList {
   constructor() {
     this.todos = getTodos();
   }
 
-  handleCompletedStatus = (e) => {
-    updateStatus(e, this.todos);
-  }
+ handleCompletedStatus = (e) => {
+   const todos = getTodos();
+   updateStatus(e, todos);
+ }
 
-  handleDropEvt = (e) => {
-    handleDrop(e, this.todos);
-  }
+ handleEditTodo = (e) => {
+   if (e.keyCode === 13) {
+     const { value } = e.target;
+     const index = parseInt(e.target.dataset.index, 10) - 1;
+     editTodoInStorage(value, index);
+     editTodoInList(value, index);
+   }
+ }
+
+ handleDeleteTodo = (e) => {
+   const index = parseInt(e.target.dataset.index, 10) - 1;
```

Comment:
- [ ] OPTIONAL: The delete button is not working at all. It throws a console error complaining about an invalid selector here. This should have been a mandatory change but since it was not mentioned by the previous reviewer I will leave it to you as an optional change. However, I strongly advise you to fix this issue as it is work that shows your skills and professionalism to potential employers. :pray: 
- [ ] OPTIONAL: In addition, deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. Try to fix this one too. :pray: 

![Screenshot from 2021-09-24 19-53-07](https://user-images.githubusercontent.com/64894927/134686432-14ead343-a900-48e1-a6eb-7def1cfdf077.png)